ceph.go

```diff
@@ -166,19 +168,24 @@ func (pqo *pgQueryOut) getCompletePeers() []int {
 			if err != nil {
 				panic(fmt.Sprintf("%s: %s in peer ID %q is not a valid index", pqo.Info.PgID, m[2], pi.Peer))
 			}
+
+			// Save the last_epoch_clean for later comparison
+			osdEpochMap[osd] = pi.Stats.LastEpochClean
```

Comment:
I don't think that's used

Reply:
Use it as on line 180 to do the epoch comparison